### PR TITLE
Update use cases

### LINKED ISSUE #9: Update DG Use Cases

### PR DESCRIPTION
**Pull request by tanjiaxian99**
Closes #9

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -79,10 +81,10 @@ The `UI` component uses the JavaFx UI framework. The layout of these UI parts ar
...
 The `UI` component,
 
-* executes user commands using the `Logic` component.
-* listens for changes to `Model` data so that the UI can be updated with the modified data.
-* keeps a reference to the `Logic` component, because the `UI` relies on the `Logic` to execute commands.
-* depends on some classes in the `Model` component, as it displays `Person` object residing in the `Model`.
+- executes user commands using the `Logic` component.
+- listens for changes to `Model` data so that the UI can be updated with the modified data.
+- keeps a reference to the `Logic` component, because the `UI` relies on the `Logic` to execute commands.
+- depends on some classes in the `Model` component, as it displays `Person` object residing in the `Model`.
```
Comment (junwei26) [inline docs/DeveloperGuide.md:87]: I think we might want to change all the Persons to Students since our entries in CSBook are students

**Review by junwei26 — CHANGES_REQUESTED** [review]
I think you might have missed changing `Person` to `Student` for the Architecture section.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (tanjiaxian99) [reply docs/DeveloperGuide.md:87]: We are not supposed to change the Architecture section in this version. My IDE just automatically made changes to some lines of code when I saved the file.

Comment (junwei26) [reply docs/DeveloperGuide.md:87]: Ok LGTM!

**Review by junwei26 — APPROVED** [review]
LGTM!

**Pull request merged**